File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        Map<Integer, Integer> freqs = new HashMap<>();
        int maxFreq = Integer.MIN_VALUE;
        int res = 0;
        for (int num : nums) {
            int f = freqs.getOrDefault(num, 0) + 1;
            freqs.put(num, f);
            if (f > maxFreq) {
                maxFreq = f;
                res = f;
            } else if (f == maxFreq) {
                res += f;
            }
        }
        return res;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        freqs = collections.Counter(nums)
        max_freq = -1
        res = 0
        for k, f in freqs.items():
            if f > max_freq:
                max_freq, res = f, f
            elif f == max_freq:
                res += f
        return res